jdbi/src/test/java/org/skife/jdbi/v2/TestContainerFactory.java

```diff
@@ -32,16 +32,16 @@
 import org.skife.jdbi.v2.tweak.ContainerFactory;
 import org.skife.jdbi.v2.util.StringMapper;
 
+import java.util.ArrayList;
 import java.util.LinkedHashSet;
+import java.util.List;
 import java.util.Set;
 import java.util.UUID;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+// FIXME-1615 : How to run this test?
```

Comment:
I think only the `@JDBITests` are run when invoking `mvn test -Pjdbi` (check the pom.xml). I think these `@JDBIQuarantineTests` tests are disabled and we could remove them now.